qa/sc_evm_smart_contract_transaction.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+import json
+import pprint
+from decimal import Decimal
+
+import test_framework.util
+from SidechainTestFramework.account.ac_use_smart_contract import SmartContract
+from SidechainTestFramework.account.mk_contract_address import mk_contract_address
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration, Account, LARGE_WITHDRAWAL_EPOCH_LENGTH
+from SidechainTestFramework.sc_forging_util import sc_create_forging_stake_mempool
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from test_framework.util import assert_equal, assert_true, start_nodes, \
+    websocket_port_by_mc_node_index, forward_transfer_to_sidechain, COIN
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, \
+    start_sc_nodes, is_mainchain_block_included_in_sc_block, check_box_balance, \
+    check_mainchain_block_reference_info, check_wallet_coins_balance, check_box_balance, get_lib_separator, \
+    AccountModelBlockVersion, EVM_APP_BINARY, generate_next_blocks, generate_next_block, generate_account_proposition, \
+    convertZenniesToWei, convertZenToZennies
+
+"""
+Check the EVM bootstrap feature.
+
+Configuration: bootstrap 1 SC node and start it with genesis info extracted from a mainchain node.
+    - Mine some blocks to reach hard fork
+    - Create 1 SC node
+    - Extract genesis info
+    - Start SC node with that genesis info
+
+Test:
+    For the SC node:
+        - verify the MC block is included
+"""
+
+
+class SCEvmBootstrap(SidechainTestFramework):
```

Comment:
need to rename the class to avoid the name collisions later, for example:
SCEvmBootstrap -> SCEvmStorageContract